rotkehlchen/accounting/structures/base.py

```diff
@@ -279,16 +280,17 @@ def process(
                 return 1
 
             # otherwise it's kraken staking
-            accounting.add_acquisition(
-                event_type=AccountingEventType.STAKING,
-                notes=f'Kraken {self.asset.symbol} staking',
-                location=self.location,
-                timestamp=self.get_timestamp_in_sec(),
-                asset=self.asset,
-                amount=self.balance.amount,
-                taxable=True,
-            )
-            return 1
+            if self.asset == A_ETH2 and accounting.settings.eth_staking_taxable_after_withdrawal_enabled is False:  # noqa: 501
```

Comment:
ehm ... no man come on ... think a bit before writing code!

This is introducing a bug.

Kraken staking can be for any asset. Not just ETH/ETH2.

So for any other asset this is not gonna create an event, which is a bug.